testsuite/python/tests_common.py

```diff
@@ -173,6 +173,31 @@ def transform_vel_from_cartesian_to_polar_coordinates(pos, vel):
         (pos[0] * vel[1] - pos[1] * vel[0]) / (pos[0]**2 + pos[1]**2), vel[2]])
 
 
+def define_rotation_matrix(system, part):
```

Comment:
1. Why is this function identical to `rotation_matrix_quat()`?
2. Can `define_rotation_matrix()`, `rotation_matrix_quat()`, `convert_vec_body_to_space()` be simplified down to the following code?
    ```python
    from scipy.spatial.transform import Rotation
    
    def rotation_matrix_quat(quat):
        return Rotation.from_quat(np.roll(quat, -1)).as_dcm()
    
    def convert_vec_body_to_space(system, part, vec):
        R = define_rotation_matrix(system[part].quat)
        return np.dot(R, vec)
    ```

Reply:
Nevermind point 2, `scipy.spatial.transform` is not available in the `scipy` version packaged in the Ubuntu 18.04 repository.

Reply:
@jngrad for point 1: agree, I will deduplicate this. Perhaps, this PR is just very old is come routines have been added.

Reply:
@jngrad point 1: done.